templates/data/mysql.html

```diff
@@ -1,477 +1,533 @@
-        </ul>
-        <p>You can also get 24/7 support with optional screen sharing for all your MySQL deployments.</p>
+      <div class="col">
+        <div class="p-section--shallow">
+          <div class="p-image-container--16-9 is-highlighted u-hide--small">
+            {{ image(url="https://assets.ubuntu.com/v1/1c4c6ee0-encryption.png",
+                        alt="",
+                        width="300",
+                        height="300",
+                        hi_def=True,
+                        attrs={"class": "p-image-container__image"},
+                        loading="lazy") | safe
+            }}
+          </div>
+        </div>
         <p>
-          <a class="p-button" href="https://ubuntu.com/pro/subscribe">Purchase your subscription</a>
+          Charmed MySQL comes with up to 10 years of security maintenance. It also ships with hardened configurations to help you comply with the most stringent requirements in terms of encryption, auditing, logging, authentication and authorisation.
         </p>
       </div>
     </div>
-  </div>
-</section>
+  </section>
 
-<section class="p-section">
-  <hr class="p-rule is-fixed-width">
-  <div class="row--50-50">
-    <div class="col">
-      <div class="p-section--shallow">
-        <h2>How it works</h2>
+  <section class="p-section u-fixed-width">
+    <hr />
+    <div class="row--50-50">
+      <div class="col">
+        <h2>MySQL support</h2>
+      </div>
+      <div class="col">
+        <div class="p-section--shallow">
+          <div class="p-image-container--16-9 is-highlighted u-hide--small">
+            {{ image(url="https://assets.ubuntu.com/v1/d3e46567-postgres-support.png",
+                        alt="",
+                        width="442",
+                        height="250",
+                        hi_def=True,
+                        attrs={"class": "p-image-container__image"},
+                        loading="lazy") | safe
+            }}
+          </div>
+        </div>
+        <p>Get comprehensive, 24/7 or weekday support for Charmed MySQL, covering the entire solution for up to 10 years.</p>
+        <hr class="is-muted" />
+        <p>
+          <a class="p-button" href="https://ubuntu.com/pro/subscribe">Subscribe now<span class="u-off-screen">Subscribe for ubuntu pro service</span></a>
+        </p>
       </div>
     </div>
-    <div class="col">
-      <p>The diagram below depicts a high level overview on the components we use. Key architecture insights:</p>
-      <div class="p-section--shallow">
-        {{ image (
-          url="https://assets.ubuntu.com/v1/f330414d-mysql-graph.png",
-          alt="",
-          width="1800",
-          height="1107",
-          hi_def=True,
-          loading="lazy"
-          ) | safe
-        }}
+  </section>
+
+  <section class="p-section">
+    <div class="row--50-50 p-section--shallow">
+      <hr />
+      <div class="col">
+        <h2>
+          Predictable pricing
```

Comment:
Please add "for"
![Screenshot 2024-09-09 at 12 01 42](https://github.com/user-attachments/assets/ce2474ac-8993-4e46-95ca-6cb2563b469e)